### DISCUSSION TITLE
Get run ID in field formula?

### DISCUSSION THREAD

**Discussion by tennox** [windmill-labs/windmill#4337] [Questions] [answered]
Can I get the flow run ID in a formula field?
![image](https://github.com/user-attachments/assets/dea7b7b2-d16f-4a44-a94c-695bd6efda3c)

**Comment by rubenfiszel** [comment, marked as answer]
you can't, at least not right now, for now you would return it from env WM_FLOW_ROOT_JOB_ID variable in one of the step and use it as a result
